src/formatting_stack/linters/eastwood/impl.clj

```diff
@@ -9,23 +9,29 @@
   "Does this linting result refer to a :pre/:post containing dynamic assertions?
 
 See https://git.io/fhQTx"
-  [{{{[_fn* [_arglist & fn-tails]] :form} :ast} :wrong-pre-post
-    msg                                         :msg
-    :as                                         x}]
-  (->> fn-tails
-       (some (fn [tail]
-               (when (and (coll? tail)
-                          (= 'clojure.core/assert
-                             (first tail)))
-                 (let [v (second tail)
-                       v-name (when (symbol? v)
-                                (name v))]
-                   (and v-name
-                        (string/includes? msg v-name) ;; make sure it's the same symbol as in msg
-                        (= \*
-                           (first v-name)
-                           (last v-name)))))))
-       (boolean)))
+  [{{{xxx :form} :ast} :wrong-pre-post
+    msg                :msg
+    :as                x}]
+  (let [[_fn* yyy] (if (coll? xxx)
+                     xxx
+                     [])
+        [_arglist & fn-tails] (if (coll? yyy)
+                                yyy
+                                [])]
+    (->> fn-tails
```

Comment:
naming thing is hard.

I think I named the fn-tails wrong before.
```clj
(fn* 
  ([] (foo 1) (bar 1))
  ([x] (foo x) (bar x)))
```

right now we call `(foo 1) (bar 1)` `"fn-tails"`, while I think `([] (foo 1) (bar 1))` is the actual fn-tail.

suggestion:
 - `fn-tails` -> `body` or `forms`
 - `xxx` -> `ast-form`
 - `yyy` -> `fn-tails`
 - `tail` -> `form`


wdty?